installers/WindowsInstaller.hs

```diff
@@ -1,32 +1,34 @@
 module WindowsInstaller where
 
-import           Data.Maybe (fromMaybe)
-import           Data.Monoid ((<>))
-import qualified Data.List as L
+import qualified Data.List          as L
+import           Data.Maybe         (fromMaybe)
+import           Data.Monoid        ((<>))
 import           Development.NSIS
-import           System.Directory (renameFile)
+import           System.Directory   (renameFile)
 import           System.Environment (lookupEnv)
-import           Turtle (echo, procs)
+import           Turtle             (echo, procs)
 
 shortcutParameters :: [String] -> String
 shortcutParameters ipdht = L.intercalate " " $
-  [ "--node \"%PROGRAMFILES%\\Daedalus\\cardano-node.exe\""
-  , "--node-log-path", "\"%APPDATA%\\Daedalus\\Logs\\cardano-node.log\""
-  , "--wallet \"%PROGRAMFILES%\\Daedalus\\Daedalus.exe\""
-  , "--updater C:/TODO" -- TODO
-  , "--node-timeout 5"
-  , (" -n " ++ (L.intercalate " -n " nodeArgs))
-  ] 
-    where
-      nodeArgs = [
-        "--listen", "0.0.0.0:12100",
-        "--log-config", "log-config-prod.yaml",
-        "--keyfile", "\"%APPDATA%\\Daedalus\\Secrets\\secret.key\"",
-        "--logs-prefix", "\"%APPDATA%\\Daedalus\\Logs\"",
-        "--db-path", "\"%APPDATA%\\Daedalus\\DB-0.2\"",
-        "--wallet-db-path", "\"%APPDATA%\\Daedalus\\Wallet-0.2\"",
-        "--wallet"
-        ] <> ("--peer" : (L.intersperse "--peer" ipdht))
+    [ "--node \"%PROGRAMFILES%\\Daedalus\\cardano-node.exe\""
+    , "--node-log-path", "\"%APPDATA%\\Daedalus\\Logs\\cardano-node.log\""
+    , "--wallet \"%PROGRAMFILES%\\Daedalus\\Daedalus.exe\""
+    , "--updater \"" <> updaterPath <> "\""
+    , "--node-log-config", "log-config-prod.yaml" -- should it be just here, not in programfiles/appdata?
```

Comment:
@domenkozar where should it point to? I think we should put a full path here, not a relative (from what dir?).